sdk/apis/apis/v1alpha2/types_apiexport_conversion.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"encoding/json"
+	"fmt"
+
+	kubeconversion "k8s.io/apimachinery/pkg/conversion"
+
+	apisv1alpha1 "github.com/kcp-dev/kcp/sdk/apis/apis/v1alpha1"
+)
+
+const (
+	resourceSchemasAnnotation = "apis.v1alpha2.kcp.io/resource-schemas"
```

Comment:
this must be internal. Then it is protected automatically. This is not meant to be user changeable, is it?

Reply:
Actually, not sure my comment makes sense. Everybody can and must be able to change it, right? A conversion happens as normal user. We must validate it to make sense.